tools/backups2datalad.py

```diff
@@ -676,6 +678,15 @@ def release(
     datasetter.mkrelease(dandiset_obj, dataset, commitish=commitish, push=push)
 
 
+@contextmanager
+def custom_commit_author() -> Iterator[None]:
+    with envset("GIT_AUTHOR_NAME", "DANDI User"):
+        with envset("GIT_AUTHOR_EMAIL", "info@dandiarchive.org"):
+            with envset("GIT_COMMITTER_NAME", "DANDI Team"):
+                with envset("GIT_COMMITTER_EMAIL", "help@dandiarchive.org"):
```

Comment:
sorry I was not exhaustive in my "Primary use case for me is to make it easier to say who is who in a given commit and relate to the dates." and specification there in.  The main point was to whenever we use date from the archive (not current date) make it clear by looking at the AUTHOR associated name to say that it was not us here to did it.

Let's completely drop tuning `GIT_COMMITTER` anything and let that be as configured on the system. For that I changed `~/.gitconfig` to
```
[user]
        name = DANDI Team
        email = help@dandiarchive.org
```

- remove this `custom_commit_author` and just make `custom_commit_date` to adjust also GIT_AUTHOR_NAME and GIT_AUTHOR_EMAIL per values above. This way any commit with "custom date" would be associated with the AUTHOR being different from the COMMITTER (per default git configuration etc). 
- for changes coming from the archive, we can't say if they were done by the actual archive "User" or by an "Admin", but we can consider those "users" anyways.